### PR TITLE
{vis}[intel/2015a] SPIDER 22.10 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/s/SPIDER/SPIDER-22.10-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+buildininstalldir = True
+
+parallel = 1
+
+skipsteps = ['configure', 'install']
+
+prebuildopts = "rm ../bin/spider* && rm ../src/spider*.[ao] && "
+buildopts = '-f Makefile_linux COMP="$F90" EXE=spider FFTWLIBDIR=$EBROOTFFTW/lib '
+buildopts += 'FFLAGS="$F90FLAGS -WB -DSP_LIBFFTW3 -DSP_LINUX -DSP_IFC -fpp -c" LF="$F90FLAGS"'
+#buildopts += 'FFLAGS="$F90FLAGS -DHAS_IEEE -DSP_LIBFFTW3 -DSP_LINUX -DSP_IFC -DSP_MP -c" LF="$F90FLAGS"'
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/SPIDER/SPIDER-22.10-intel-2015a.eb]: why the comment?

Comment (boegel) [inline easybuild/easyconfigs/s/SPIDER/SPIDER-22.10-intel-2015a.eb]: ah, this is leftover crap, thanks;

I need to look into the `-mp` and `-mpi` variants too, forgot about that...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/46f2a2d447b258b260cb for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/bdf8de24c558243339af for a full test report.

**Comment by boegel** [comment]
mpi build is still broken...

**Comment by boegel** [comment]
Now includes `-mt` and `-mpi` variants, which required more updates to the patch file.

@wpoely86: please rerereview and retest?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/19e3bb26be757790ea5a for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 4 out of 4 (3 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/a7340de4c949afe8bab8 for a full test report.

**Comment by boegel** [comment]
Thanks for the review/test @wpoely86!

**Pull request merged**